### PR TITLE
Python SDK

### PR DESCRIPTION
**Pull request by nsosio**
- [x] `prem` package coverage
  - [x] client interface
  - [x] completions
  - [x] embeddings
  - [x] datapoints
- [x] publish prem package on tag 
- [x] sphinx documentation
- [x]  publishing documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,107 @@
...
+    source venv/bin/activate
+    pip install -r requirements.txt
+    pip install .
+    ```
+## Usage
+### Getting Started
+To use the Prem Python SDK, you need to obtain an API key from the Prem platform. You can then create a `PremAI` instance to make requests to the API.
+
+```python
+from prem import PremAI
```
Comment (filopedraz) [inline README.md]: Prem instead of PremAI

File: prem/client.py [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+from typing import Dict, List, Union
+
+import requests
+from sseclient import SSEClient
+
+from . import resources
+from .exceptions import error_to_exception
+
+
+class PremAI:
```
Comment (filopedraz) [inline prem/client.py]: Same

**Comment by filopedraz** [comment]
We need to publish the package on `pipy` as soon as we create a new tag.

File: test.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+from prem import PremAI
+
+api_key = "3EC9HA3EFaQgFbzE8TqsOb0OTrTfFxv2x9"
+base_url = "http://localhost:8000"
```
Comment (filopedraz) [inline test.py:6]: We should have a default base_url pointing to prod

File: README.md (hunk 1 above)
Comment (nsosio) [reply README.md]: Done  :)

File: prem/client.py (hunk 2 above)
Comment (nsosio) [reply prem/client.py]: Donee :)

File: test.py (hunk 3 above)
Comment (nsosio) [reply test.py:6]: what's the prod url?

Comment (filopedraz) [reply test.py:6]: https://app.prod.prem.ninja/

Comment (filopedraz) [reply test.py:6]: But we probably need to change it to premai.io

**Comment by filopedraz** [comment]
For the last two points, we can handle them tomorrow :) Let me know if I need to create an account or anything.

File: test.py (hunk 3 above)
Comment (nsosio) [reply test.py:6]: done

**Review by giowe — APPROVED** [review]

**Pull request merged**